packages/browser/__tests__/login/oidc/redirectHandler/ErrorOidcHandler.spec.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2021 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { jest, it, describe, expect } from "@jest/globals";
+import { ErrorOidcHandler } from "../../../../src/login/oidc/redirectHandler/ErrorOidcHandler";
+
+describe("ErrorOidcHandler", () => {
+  describe("canHandle", () => {
+    it("always accept the given IRI", async () => {
```

Comment:
```suggestion
    it("doesn not handle IRIs without an error parameter", async () => {
```
The test description typically summarizes what requirement the test verifies, so that test reports provide as much context as possible when a test fails.